src/App.js

```diff
@@ -1,17 +1,17 @@
-import React, { Component } from 'react';
-import { ApolloProvider } from 'react-apollo';
+import React, { Component } from "react";
+import { ApolloProvider } from "react-apollo";
+import { BrowserRouter as Router } from "react-router-dom";
```

Comment:
super minor and logging it before I forget, but I think the prettier you're using isn't using the settings in package.json -- I think that's why my last PR merge made everything single quotes, and yours is turning everything back to double quotes. definitely NBD.